src/fft-api/pickjob/fftPickJobService.ts

```diff
@@ -122,4 +126,128 @@ export class FftPickJobService {
       throw err;
     }
   }
+
+  public async getAll(
```

Comment:
ich würde vorschlagen, hier ein parameter Objekt einzuführen: `getAll(params: SomeTypeName)`
und in `SomeTypeName` stehen dann all diese möglichen optionalen Parameter
das macht die Benutzung einfacher
sonst sieht der Aufruf nachher so aus `getAll(undefined, undefined, undefined, orderId)

Reply:
Guter Punkt, habe das interface PickJobsQueryParameters hinzugefügt.